recipes/flask-restx/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "flask-restx" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "bf9885bffe96ead2d5a7f53000a1d3562bf0cceca4f4a26c5d64d187aee12a25" %}
+{% set requirements = ["aniso8601 >=0.82", "jsonschema", "Flask >=0.8", "pytz", "six >=1.3.0", "enum34"] %}
```

Comment:
FWIW This is a very strange way to do this and may not be compatible with our bots

Reply:
Hmm, what is wrong with it? I just want to be able to put a data from requirements.txt file to the receipt. It is easier to replace a one line

Reply:
I'm going to implement a script which will read the requirements.txt file, convert it to a list and then just replace one line in the recipe. Thi script will be triggered for every new release, it all should update the version and sha256.

Reply:
For example, I am not certain that the linter bot will correctly lint recipes in this form.  Similarly, the auto-tick bot may or may nor be able to migrate dependencies written like this.  It really just depends on whether the bot tool reads the raw meta.yaml or the rendered meta.yaml

Reply:
So can we check it? I would like to keep
I think auto-tick bot should read the rendered recipe, otherwise it will not know the recipe's name

Reply:
The auto-tick bot does a lot of different things depending.  The code for it is at https://github.com/regro/cf-scripts/. 

The code for the linter is at https://github.com/conda-forge/conda-smithy

Reply:
auto-tick bot:
I found only one place where it rear the raw meta.yml file: https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators/arch.py#L92
It reads a build number from the file.
Other code use render function.


linter also use render function.

Reply:
Can it break anything if we will merge it? If anything wrong, I will update the recipe

Reply:
anyway linter is green

Reply:
It will probably only break this package:) If you are comfortable with fixing it if need be, I am happy to merge it!

Reply:
Yeah, I'm ok to fix it if it will be broken. Let's merge

Reply:
Thank you